pkg/fixtures/triggers/reporting.report_run.succeeded.json

```diff
@@ -0,0 +1,19 @@
+{
+  "_meta": {
+    "template_version": 0
+  },
+  "fixtures": [
+    {
+      "name": "reporting",
+      "path": "/v1/reporting/report_runs",
+      "method": "post",
+      "params": {
+        "report_type":"balance.summary.1",
+        "parameters": {
+          "interval_start": 1641045832,
+          "interval_end": 1646143432,
+        },
```

Comment:
I'm getting an error when I run this because the json parser does not expect these two commas at the end of the list, are you seeing the same thing?

Reply:
Hi @gracegoo-stripe I have addressed the two points you brought up. Could you review this again, please? Thank you.